x/twap/store_test.go

```diff
@@ -162,11 +162,11 @@ func (s *TestSuite) TestGetRecordAtOrBeforeTime() {
 	defaultInputAt := func(t time.Time) getRecordInput { return getRecordInput{1, t, denom0, denom1} }
 	wrongPoolIdInputAt := func(t time.Time) getRecordInput { return getRecordInput{2, t, denom0, denom1} }
 	defaultRevInputAt := func(t time.Time) getRecordInput { return getRecordInput{1, t, denom1, denom0} }
-	baseRecord := newEmptyPriceRecord(1, baseTime, denom0, denom1)
+	baseRecord := withPrice0Set(newEmptyPriceRecord(1, baseTime, denom0, denom1), sdk.OneDec())
 	tMin1 := baseTime.Add(-time.Second)
-	tMin1Record := newEmptyPriceRecord(1, tMin1, denom0, denom1)
+	tMin1Record := withPrice0Set(newEmptyPriceRecord(1, tMin1, denom0, denom1), sdk.OneDec())
 	tPlus1 := baseTime.Add(time.Second)
-	tPlus1Record := newEmptyPriceRecord(1, tPlus1, denom0, denom1)
+	tPlus1Record := withPrice0Set(newEmptyPriceRecord(1, tPlus1, denom0, denom1), sdk.OneDec())
```

Comment:
why are we now setting price0 instead of initializing with empty price record?

Reply:
price0 of 0 would panic because the logarithm of 0 is undefined.

This is safe because, logically, spot price of 0 should be undefined as well